.github/workflows/basic-ci-amd64.yaml

```diff
@@ -22,8 +22,8 @@ jobs:
       fail-fast: false
       matrix:
         containerd_version: ['lts', 'active']
-        vmm: ['clh', 'dragonball', 'qemu', 'stratovirt', 'cloud-hypervisor']
-    runs-on: garm-ubuntu-2204-smaller
+        vmm: ['clh', 'dragonball', 'qemu', 'stratovirt']
+    runs-on: ubuntu-22.04
```

Comment:
FYI GHA won't pick up such YAML changes in this PR because it's running YAMLs from the target branch, not the base branch.